examples/plotting/file/eclipse.py

```diff
@@ -0,0 +1,75 @@
+# Based on https://www.reddit.com/r/dataisbeautiful/comments/6qnkg0/google_search_interest_follows_the_path_of_the/
+
+import shapefile as shp
+import pandas as pd
+
+from bokeh.plotting import figure, show
+from bokeh.palettes import YlOrRd5
+from bokeh.models import ColumnDataSource, LinearColorMapper, Label, ColorBar
+from bokeh.sampledata.us_states import data
+
+states = pd.DataFrame.from_dict(data, orient="index")
+states.drop(["AK", "HI"], inplace=True)
+
+trends = pd.read_csv("eclipse_data/trends.csv")
+
+states.set_index("name", inplace=True)
+trends.set_index("Region", inplace=True)
+
+states["trend"] = trends["solar eclipse"]
+
+upath17 = shp.Reader("eclipse_data/upath17")
+(totality_path,) = upath17.shapes()
+
+p = figure(plot_width=1000, plot_height=600)
+
+p.grid.grid_line_color = None
+
+p.title.text = "Google Search Trends and the Path of Solar Eclipse, 21 August 2017"
+p.title.align = "center"
+p.title.text_font_size = "16pt"
+
+mapper = LinearColorMapper(palette=list(reversed(YlOrRd5)), low=0, high=100)
+
+source = ColumnDataSource(data=dict(
+    state_xs=list(states.lons),
```

Comment:
I'm not sure what's going on here, but if I remove `list()`, then I get:
```
ValueError: Out of range float values are not JSON compliant: nan
```

Reply:
@bryevdv, do you have any immediate idea what's going on here?

Reply:
no but it will have to wait, suggest another issue

Reply:
Sure. This "out of range float" has been popping out here and there for a while now.